pre_commit_hooks/loaderon_hooks/tests/general_hooks/check_using_pylint_test.py

```diff
@@ -0,0 +1,46 @@
+import os
+import sys
+import sys
+import unittest
+
+import mock
+import pytest
+
+from pre_commit_hooks.loaderon_hooks.general_hooks.check_using_pylint import main
+from pre_commit_hooks.loaderon_hooks.tests.util.test_helpers import perform_test_on_file_expecting_result
+
+
+@pytest.fixture(autouse=True)
+def clean_sys_argv():
+    sys.argv = []
+
+    # What files should we exclude from pylint checker
+    sys.argv.append('--exclude')
+    sys.argv.append(r'.*(\/)*incorrect_but_ignored.py')
```

Comment:
I would think that `pre-commit`'s built in excludsion would work better than reimplementing it in a specific hook (that's a lot of the reason I built the framework is because various tools do inclusion / exclusion really poorly)